src/components/CCIP/ChainHero/ChainHero.css

```diff
@@ -36,7 +36,7 @@
 }
 
 .ccip-chain-hero__content {
-  --gutter: var(--space-10x);
```

Comment:
Spacing in this file are wrong (not particularly in this line of code).

It should be 24 px between the headings and 8 between label and text:
![image](https://github.com/user-attachments/assets/7ca339d0-81fd-4e2b-b13e-a033d7114e43)